index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
Add a FIXME for placing the correct language code when doing i18n. (I will also add it to the active i18n branch too.)

Reply:
Good call. Added in 994a5fd0e1e5bb59571ea28c81b8f951e9f9a482

Reply:
I'd be interested to see how this applies to a website that can change languages, but I guess it's good enough to have the fixme there. Maybe just add a note that the `lang` attribute is there to make a11y happy.

Reply:
FYI I have added the code to update the `lang` attribute in https://github.com/opentripplanner/otp-react-redux/pull/408/commits/01797599815ddde031061e9908abd267156c020b in the #408 branch.